XtermBlazor.Demo.Server/Pages/Index.razor

```diff
@@ -157,6 +158,11 @@ EventCallback Logs:
         await _terminalEvent.WriteLine($"({++_eventId}) OnKey(): Key: {args.Key}, Ctrl: {args.CtrlKey} {JsonSerializer.Serialize(args)}");
     }
 
+    private async Task OnKeyDataEvent(string key)
+    {
+        await _terminalEvent.WriteLine($"({++_eventId}) OnKeyDataEvent(): Key: {key}, Unicode: {UnicodeHandler.StringToUnicode(key)}");
```

Comment:
Added Unicode here because key will be whitespace if xterm returns only a string of unicodes